index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <div id="content">
+        <link rel="stylesheet" href="index.css">
+        <meta charset="UTF-8">
+        <meta name="descrption" content="Bening's Porfolio">
+        <title>Joe Johnson</title>
+       <div style="text-align: center">
+        <a href="portfolio.html" style="text-align:center"><em><strong> About me</em></strong></a> |
+           <a href="index.html" style="text=align:center"><strong>Porfolio</strong></a>
+             <h1>Joe Johnson</h1>
```

Comment:
@bening365 great use of indentation, just be consistent all across your code. :thumbsup: